@shared/api/external.ts

```diff
@@ -95,6 +97,33 @@ export const submitBlob = async (
   return signedBlob;
 };
 
+export const submitAuthEntry = async (
+  entryXdr: string,
+  opts?: {
+    accountToSign?: string;
+  },
+): Promise<string> => {
+  let response = { signedAuthEntry: "", error: "" };
+  const _opts = opts || {};
+  const accountToSign = _opts.accountToSign || "";
+  try {
+    response = await sendMessageToContentScript({
+      entryXdr,
+      accountToSign,
+      type: EXTERNAL_SERVICE_TYPES.SUBMIT_AUTH_ENTRY,
+    });
+  } catch (e) {
+    console.error(e);
+  }
+  const { signedAuthEntry, error } = response;
+
+  if (error) {
+    throw error;
+    throw error;
```

Comment:
repeated line

Reply:
whops, thanks for the catch. Fixed in 4b0d05a220d333be1420ba3a45be60475512e133